robotoff/workers/tasks/import_image.py

```diff
@@ -215,6 +227,68 @@ def run_nutrition_table_object_detection(product_id: ProductIdentifier, image_ur
 }
 
 
+def run_upc_detection(product_id: ProductIdentifier, image_url: str):
+    """Detect the presence of a UPC in an image and determine if it dominates the image (i.e. if it is the primary content in the image)
+    :param product_id: identifier of the product
+    :param image_url: URL of the image to use
+    """
+    image = get_image_from_url(image_url, error_raise=False, session=http_session)
+
+    if image is None:
+        logger.info("Error while downloading image %s", image_url)
+        return
+
+    open_cv_image = numpy.array(image)
+
+    source_image = get_source_from_url(image_url)
+
+    pct_area, prediction_class, box = find_image_is_upc(
+        open_cv_image
+    )  # run upc detection
+
+    with db:
+        imagemodel = ImageModel.get_or_none(
```

Comment:
`imagemodel` -> `image_model`, for consistency with the rest of the code

Reply:
Done!